pkg/utils/scheduled/utils.go

```diff
@@ -0,0 +1,72 @@
+package scheduled
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/robfig/cron"
+
+	shpmetav1 "github.com/universityofadelaide/shepherd-operator/pkg/apis/meta/v1"
+)
+
+// GetScheduledTime from an annotation map.
+func GetScheduledTime(annotations map[string]string) (*time.Time, error) {
```

Comment:
I'm guessing these were ripped out of one of the deleted deps?

Reply:
Correct - everything in these `utils` directory is from skpr/operator (RIP).